packages/itwinui-css/src/input/input.scss

```diff
@@ -1,70 +1,76 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 @import '../utils/index';
 
 @mixin iui-input {
+  --_iui-input-background-color: var(--iui-color-background-1);
+  --_iui-input-border-color: var(--iui-color-foreground-4);
+  --_iui-input-text-color: var(--iui-color-foreground-2);
+
   @include iui-reset;
   @include iui-focus($offset: -2px, $thickness: 2px);
   width: 100%;
   font-family: inherit;
-  font-size: $iui-font-size;
-  font-weight: $iui-font-weight-normal;
-  line-height: $iui-line-height;
-  border-radius: $iui-border-radius;
+  font-size: var(--iui-font-size-1);
+  font-weight: var(--iui-font-weight-normal);
+  line-height: var(--iui-size-l);
+  border-radius: var(--iui-border-radius-1);
   appearance: none;
-  padding: $iui-component-padding-vertical $iui-component-padding-horizontal;
-  min-height: $iui-component-height;
-  color: var(--iui-text-color);
-  background-color: var(--iui-color-background-1);
-  border: 1px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
-  @media (prefers-reduced-motion: no-preference) {
-    transition: border-color $iui-speed-fast ease-out;
-  }
-
-  &.iui-small {
-    padding-top: $iui-component-padding-vertical-small;
-    padding-bottom: $iui-component-padding-vertical-small;
-    min-height: $iui-component-height-small;
-  }
-
-  &.iui-large {
-    padding-top: $iui-component-padding-vertical-large;
-    padding-bottom: $iui-component-padding-vertical-large;
-    min-height: $iui-component-height-large;
-    font-size: $iui-font-size-leading;
-  }
+  min-height: var(--_iui-input-min-height);
+  padding-block: var(--_iui-input-padding-block);
+  padding-inline: var(--iui-size-s);
+  color: var(--_iui-input-text-color);
+  background-color: var(--_iui-input-background-color);
+  border: 1px solid var(--_iui-input-border-color);
+  transition: border-color var(--iui-duration-1) ease-out;
+  box-sizing: border-box;
```

Comment:
I think these were again done by Jon. I've removed the `box-sizing` property because it's now set globally, but I've kept the removal of media query as it's not necessary.